18_regexp_lab/01.py

```diff
@@ -1,12 +1,12 @@
-"""
-Write a program that reads data
-from property files.
-Each line in the file can either be:
-    An empty line
-    A comment line (Start with #)
-    A property line (of the form key = value)
-
-Write a program that takes a property file name and key
-as command line arguments and prints the requested value
-"""
-
+import sys

```

Comment:
Please leave the top comments in the file (the ones inside """). They make it easier for everyone opening the file later to understand what's inside